src/sql/platform/dialog/dialogTypes.ts

```diff
@@ -21,16 +22,14 @@ export class DialogTab implements sqlops.window.modelviewdialog.DialogTab {
 }
 
 export class Dialog implements sqlops.window.modelviewdialog.Dialog {
+	private static readonly DONE_BUTTON_LABEL = localize('dialogModalDoneButtonLabel', 'Done');
+	private static readonly CANCEL_BUTTON_LABEL = localize('dialogModalCancelButtonLabel', 'Cancel');
+
 	public content: string | DialogTab[];
-	public okTitle: string;
-	public cancelTitle: string;
+	public okButton: DialogButton = new DialogButton(Dialog.DONE_BUTTON_LABEL, true);
```

Comment:
Minor: sometimes people want to override the "Done" label. But this is fine for now